config/initializers/constants.rb

```diff
@@ -199,6 +199,9 @@ class Constants
   # Organization name for default admin user
   DEFAULT_PRIVATE_ORG_NAME = 'My projects'.freeze
 
+  # Task widget types. Order is important, new widget types must be appended!
+  WIDGET_TYPES = %w(protocol results activities samples).freeze
```

Comment:
That was my question, isn't the point of freezing variables to make them unmodifiable :smile:?